examples/patient-hub/src/main/ml-modules/root/custom-modules/custom/BuildAdmissions/main.sjs

```diff
@@ -0,0 +1,89 @@
+/*
+  Copyright 2012-2019 MarkLogic Corporation
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+const DataHub = require("/data-hub/5/datahub.sjs");
+const datahub = new DataHub();
+
+function main(content, options) {
+
+  let outputFormat = options.outputFormat ? options.outputFormat.toLowerCase() : datahub.flow.consts.DEFAULT_FORMAT;
+
+  let doc = content.value;
+  if (doc && (doc instanceof Document || doc instanceof XMLDocument)) {
+    doc = fn.head(doc.root);
+  }
+
+  let instance = {"Admissions" : datahub.flow.flowUtils.getInstance(doc).toObject() } || {};
+  let triples = datahub.flow.flowUtils.getTriples(doc) || [];
+  let headers = datahub.flow.flowUtils.getHeaders(doc) || {};
+
+  instance['$attachments'] = doc.envelope.instance;
+
+
+   //let's assemble our admissions here, first we grab the patient and admission ID
+  let patientID = instance.Admissions.PatientID;
+  let admissionID = instance.Admissions.AdmissionID;
+
+  //now, we search for the labs that match this patient ID and admission ID, then add those
+  let diagnosesDocs = cts.search(cts.andQuery([cts.jsonPropertyRangeQuery('PatientID', '=', patientID), cts.jsonPropertyRangeQuery('AdmissionID', '=',admissionID),cts.collectionQuery(['DiagnosesCore'])]))
```

Comment:
This query is important for a user to understand, and my experience has been that it's easier to understand cts queries when they're formatted with one query per line, e.g.

```
let diagnosesDocs = cts.search(
    cts.andQuery([
      cts.jsonPropertyRangeQuery('PatientID', '=', patientID),
      cts.jsonPropertyRangeQuery('AdmissionID', '=',admissionID),
      cts.collectionQuery(['DiagnosesCore'])
    ])
  )
```

This is what the structure looks like:

```
"diagnoses": [
{
"Diagnoses": {
"primaryDiagnosisCode": "F16.151", 
"primaryDiagnosisDescription": "Hallucinogen abuse with hallucinogen-induced psychotic disorder with hallucinations"
}
}
]
```

Since every other property is upper camel case, I think it makes sense for the array property to be named "Diagnoses", and then each diagnosis has a name of "Diagnosis".

Reply:
I can make this change in the module. Admissions entity has lower case diagnoses as are other properties and so the instance comes up like that. Do you want me to make all the entity properties also Upper-Camel-Case

Reply:
Was able to make the suggested change.